### DISCUSSION TITLE
How to run Personal CI?

### DISCUSSION THREAD

**Discussion by niharrathod** [apache/pulsar#19843] [Q&A] [answered]
I followed instruction provided : https://pulsar.apache.org/contribute/personal-ci/. and raised PR on my personal fork, but it didn't started CI process. 

Issue : [#19838]
PR on personal Fork : [link](https://github.com/niharrathod/pulsar/pull/1) 
PR on apache/pulsar repo: [#19842] 

Am I missing something?

**Comment by tisonkun** [comment]
Interesting. It seems like a GitHub issue that it cannot identify GitHub Actions workflow files correctly. Sometimes I found similar case on new repos.

I suggest you add a sample workflow for verifying if it's a GitHub issue.

**Reply by niharrathod** [reply]
Ok. Thanks.

**Comment by niharrathod** [comment, marked as answer]
@tisonkun : Thanks for the quick response. :+1: 

I need to enable GitHub action on my personal fork. Phew!

**Reply by tisonkun** [reply]
Good to know :D
